### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by besheajj**
This is a first draft of the BCO

### CODE REVIEW

File: Student_BCOs/JuneBeshea_BCO_000550.json [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                "email": "mmadrid@gwu.edu"
+            },
+            {
+                "name": "Ling Sheng Loh",
+                "orcid": "https://orcid.org/0000-0003-0981-7984",
+                "affiliation": "George Washington University ",
+                "contribution": [
+                    "contributedBy"
+                ],
+                "email": "lingsloh@gwmail.gwu.edu"
```
Comment (jpat1546) [inline Student_BCOs/JuneBeshea_BCO_000550.json:37]: Your group members should not be contributors to your BCO. Creating the BCO should have been independently done.

File: Student_BCOs/JuneBeshea_BCO_000550.json [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    "bco_id": "http://biocomputeobject.org/BCO_000550",
+    "checksum": "274de4a0875b62abc9095a6669fb7b57d079dd557aa78fc84a595167cfccae25",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Correlating miRNA and gene expression data in 34 cancer phenotypes",
+        "version": "1.0.0",
+        "license": "Creative Commons Attribution 4.0 International License",
+        "created": "2019-10-17T02:13:35.166914",
+        "modified": "2019-10-17T03:16:35.853294",
+        "contributors": [
```
Comment (jpat1546) [inline Student_BCOs/JuneBeshea_BCO_000550.json:11]: Please add the review status. See #148

File: Student_BCOs/JuneBeshea_BCO_000550.json [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                "affiliation": "George Washington University ",
+                "contribution": [
+                    "contributedBy"
+                ],
+                "email": "lingsloh@gwmail.gwu.edu"
+            }
+        ]
+    },
+    "usability_domain": [
+        "This workflow will facilitate the integration of already provided with TCGA data of mRNAs, miRNAs and or proteins in cancer, from there it will calculate the correlations the miRNA and expression data. This will allow better understanding of miRNA roles in expression profiles in cancer. This example pipeline was created based on the work of Ahmed et al. doi: 10.1186/s13104-018-3160-9"
```
Comment (jpat1546) [inline Student_BCOs/JuneBeshea_BCO_000550.json:42]: After reading the publication, did you write this content yourself?

**Review by jpat1546 — CHANGES_REQUESTED** [review]
@besheajj 
I've made several comments on this draft. I'd like to discuss this BCO further tomorrow. 
When can you drop by? I will be in Rm 541 in Ross Hall.
Reassigning this pull request to you until the changes are made and we've discussed.

File: Student_BCOs/JuneBeshea_BCO_000550.json (hunk 1 above)
Comment (jpat1546) [reply Student_BCOs/JuneBeshea_BCO_000550.json:37]: Publication authors should be listed as contributors. See #113

File: Student_BCOs/JuneBeshea_BCO_000550.json (hunk 3 above)
Comment (besheajj) [reply Student_BCOs/JuneBeshea_BCO_000550.json:42]: I read the publication, I thought this was supposed to align with group members! If not I will erase and write a new one.

**Pull request closed without merging**